plutus-tx-plugin/test/Budget/allCheap.plc.golden

```diff
@@ -2,66 +2,33 @@ letrec data (List :: * -> *) a | Nil_match where
-                                            {m} dMonoid {m}
-                                            (\(v : (\a -> a -> a -> a) m) (v
-                                              : m) ->
-                                               v))
-                 (\(x : a) (xs : List a) ->
-                    /\dead ->
-                      Monoid_match
-                        {m} dMonoid {(\a -> a -> a -> a) m}
-                        (\(v : (\a -> a -> a -> a) m) (v : m) -> v) (eta x)
-                        (fFoldableNil_cfoldMap {m} {a} dMonoid eta xs))
-                 {all dead. dead}
-in
 let data Bool | Bool_match where
       True : Bool
       False : Bool
-    data (MultiplicativeMonoid :: * -> *) a | MultiplicativeMonoid_match where
-      CConsMultiplicativeMonoid
-      : (\a -> a -> a -> a) a -> a -> MultiplicativeMonoid a
-in fFoldableNil_cfoldMap
-     {(\a -> a) Bool} {integer} (let !v : MultiplicativeMonoid Bool
-                                       = CConsMultiplicativeMonoid
-                                           {Bool} (\(l : Bool) (r : Bool) ->
-                                                     Bool_match
-                                                       l {all dead. Bool}
-                                                       (/\dead -> r) (/\dead ->
-                                                                        False)
-                                                       {all dead. dead}) True
-                                 in CConsMonoid
-                                      {(\a -> a) Bool}
-                                      (\(eta : (\a -> a) Bool) (eta
-                                        : (\a -> a) Bool) ->
-                                         MultiplicativeMonoid_match
-                                           {Bool} v {(\a -> a -> a -> a) Bool}
-                                           (\(v : (\a -> a -> a -> a) Bool) (v
-                                             : Bool) ->
-                                              v) eta eta)
-                                      (MultiplicativeMonoid_match
-                                         {Bool} v {Bool}
-                                         (\(v : (\a -> a -> a -> a) Bool) (v
-                                           : Bool) ->
-                                            v))) (\(v : integer) ->
-                                                    ifThenElse
-                                                      {Bool}
-                                                      (lessThanEqualsInteger
-                                                         1 v) False True)
-     ((let a = List integer in \(c : integer -> a -> a) (n : a) ->
-                                 c
-                                   1 (c
-                                        2 (c
-                                             3 (c
+in
+letrec !go : List integer -> Bool
+         = \(ds : List integer) ->
```

Comment:
prettyprinting improvement: always put the binding on the next line so we can indent it just 2 instead of 7 for `letrec`